reference/functions/string.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: string
+published: true
+tags: [reference, functions, string]
+---
+
+[%CFEngine_function_prototype(arg)%]
+
+**Description:** Convert `arg` to string.
+
+[%CFEngine_function_attributes(arg)%]
+
+If `arg` is a container reference it will be serialized to a string.
+The reference must be indicated with `@(some_container)`.
+Strings are *not* interpretted as references.
+
+```cfegine3
+bundle agent main
```

Comment:
I think this inline example can be deleted, you have the example from core that will get pulled in.